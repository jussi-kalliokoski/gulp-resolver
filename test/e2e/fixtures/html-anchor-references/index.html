<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
    <a href="cat.jpg">Picture of a Cat</a>
</body>
</html>
