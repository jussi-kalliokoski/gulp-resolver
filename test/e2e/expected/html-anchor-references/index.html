<!DOCTYPE html>
<html lang="en">
<head>
</head>
<body>
    <a href="cat.4890123.jpg">Picture of a Cat</a>
</body>
</html>
